sympy/physics/mechanics/kane.py

```diff
@@ -188,16 +239,16 @@ def __init__(self, frame, q_ind, u_ind, kd_eqs=None, q_dependent=None,
     def _initialize_vectors(self, q_ind, q_dep, u_ind, u_dep, u_aux):
         """Initialize the coordinate and speed vectors and sets the private
         class attributes that contain them. The results are publicly accessible
-        through the attributes described in the class docstring.
-
-        """
+        through the attributes described in the class docstring."""
 
         # Initialize generalized coordinates
         q_dep = _none_handler(q_dep)
         if not iterable(q_dep):
-            raise TypeError('Dependent coordinates must be an iterable.')
+            msg = 'Dependent generalized coordinates must be an iterable.'
+            raise TypeError(msg)
         if not iterable(q_ind):
-            raise TypeError('Generalized coordinates must be an iterable.')
+            msg = 'Independent generalized coordinates must be an iterable.'
+            raise TypeError(msg)
```

Comment:
Was moving the message to a separate variable to keep the line < 80 characters? My Github viewer doesn't show 80 characters so I can't tell just looking at it.